script.js

```diff
@@ -1,406 +1,318 @@
       "In a dystopian future, Katniss Everdeen becomes a symbol of rebellion when she volunteers to take her sister's place in the brutal Hunger Games.",
-    image: './books-images/unknown.jpg'
+    image: "./books-images/the-hunger-games.jpg",
   },
   {
-    title: 'The Girl with the Dragon Tattoo',
-    author: 'Stieg Larsson',
+    title: "The Girl with the Dragon Tattoo",
+    author: "Stieg Larsson",
     year: 2005,
-    genre: 'Mystery',
+    genre: "Mystery",
     rating: 4.1,
     description:
-      'A gripping mystery novel featuring investigative journalist Mikael Blomkvist and the enigmatic hacker Lisbeth Salander.',
-    image: './books-images/unknown.jpg'
+      "A gripping mystery novel featuring investigative journalist Mikael Blomkvist and the enigmatic hacker Lisbeth Salander.",
+    image: "./books-images/the-girl-with-the-dragon-tattoo.jpg",
   },
   {
-    title: 'The Road',
-    author: 'Cormac McCarthy',
+    title: "The Road",
+    author: "Cormac McCarthy",
     year: 2006,
-    genre: 'Dystopian',
+    genre: "Dystopian",
     rating: 4,
     description:
       "Set in a post-apocalyptic world, it follows a father and son's harrowing journey to survive and find safety.",
-    image: './books-images/unknown.jpg'
+    image: "./books-images/the-road.webp",
   },
   {
     title: "The Hitchhiker's Guide to the Galaxy",
-    author: 'Douglas Adams',
+    author: "Douglas Adams",
     year: 1979,
-    genre: 'Science Fiction',
+    genre: "Science Fiction",
     rating: 4.35,
     description:
       "A comedic science fiction series that follows the misadventures of Arthur Dent after Earth's destruction.",
-    image: './books-images/unknown.jpg'
+    image: "./books-images/the-hitchhikers-guide-to-the-galaxy.webp",
   },
   {
-    title: 'The Giver',
-    author: 'Lois Lowry',
+    title: "The Giver",
+    author: "Lois Lowry",
     year: 1993,
-    genre: 'Dystopian',
+    genre: "Dystopian",
     rating: 4.12,
     description:
-      'A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.',
-    image: './books-images/unknown.jpg'
+      "A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.",
+    image: "./books-images/the-giver.webp",
+  },
+];
+
+/////////////////////Dom Objects/////////////////////
+const searchInput = document.getElementById("search-txt");
+const searchBtn = document.getElementById("search-btn");
+const allBtn = document.querySelector(".all");
+const filterBtns = document.querySelectorAll(".filter-btn");
+const sortBtns = document.querySelectorAll(".sort-btn");
+const subHeading = document.querySelector("h2");
+const cardCollection = document.getElementById("card-grid");
+const randomBtn = document.getElementById("random");
+
+/////////////////////Functions///////////////////////
+// Function that accepts a books array and show books
+const showBooks = arr => {
```

Comment:
In my opinion, the code is more readable when variables/parameters are more descriptive and perhaps not shortened.